specification/keyvault/resource-manager/readme.md

```diff
@@ -50,6 +50,18 @@ input-file:
 - Microsoft.KeyVault/preview/2018-02-14-preview/providers.json
 ```
 
+### Tag: package-preview-2020-04
```

Comment:
Please use the format as you see above. OpenAPI Hub may create this format, but it's inconsistent with all over services I looked at.

Reply:
Hi, Please find Open API Hub link below;
https://portal.azure-devex-tools.com/branch/riteshchitlangia1/azure-rest-api-specs/dev-keyvault-Microsoft.KeyVault-2020-04-01-preview...master/